tests/python/test_sparse_matrix.py

```diff
@@ -4,17 +4,21 @@
 from tests import test_utils
 
 
-@pytest.mark.parametrize('dtype', [ti.f32, ti.f64])
+@pytest.mark.parametrize('dtype, storage_format', [(ti.f32, 'col_major'),
+                                                   (ti.f32, 'row_major'),
+                                                   (ti.f64, 'col_major'),
+                                                   (ti.f64, 'row_major')])
 @test_utils.test(arch=ti.cpu)
-def test_sparse_matrix_builder_deprecated_anno(dtype):
+def test_sparse_matrix_builder_deprecated_anno(dtype, storage_format):
     n = 8
     Abuilder = ti.linalg.SparseMatrixBuilder(n,
                                              n,
                                              max_num_triplets=100,
-                                             dtype=dtype)
+                                             dtype=dtype,
+                                             storage_format=storage_format)
 
     @ti.kernel
-    def fill(Abuilder: ti.types.sparse_matrix_builder()):
+    def fill(Abuilder: ti.linalg.sparse_matrix_builder()):
```

Comment:
```suggestion
    def fill(Abuilder: ti.types.sparse_matrix_builder()):
```
This shouldn't be changed.